models/channel/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 commercetools GmbH
```

Comment:
2022?

Reply:
Yeah I wanted to do this as a follow up. Thanks for the reminder.